.cirrus.yml

```
@@ -131,6 +129,7 @@ task:
   install_python_script:
     - choco install --yes --no-progress python3 --version=3.9.6
     - python -VV
+    - pip install zmq
```

Maybe
```suggestion
    - pip install zmq
    - python -VV
```
to easy spot the installed python version?